frontend/src/components/Overlay/OverlayHeader.css

```diff
@@ -3,20 +3,22 @@
 	justify-content: space-between;
 	align-items: center;
 	height: 5rem;
+	margin: 0;
+	padding: 0 1em;
 	border-top-left-radius: 0.625rem;
 	border-top-right-radius: 0.625rem;
 	background-color: var(--overlay-header-background-colour);
 }
 
 .overlay-header h1 {
-	padding: 0 2rem;
+	margin: 0;
 }
 
 .overlay-header button {
 	display: flex;
 	gap: 0.5rem;
 	align-items: center;
-	margin-right: 1rem;
+	margin: 0;
```

Comment:
Buttons are `margin: 0` by default, unless we are inheriting margin from somewhere else.